httpx_oauth/clients/kakao.py

```diff
@@ -0,0 +1,63 @@
+</svg>
+"""
+
+
+class KakaoOAuth2(BaseOAuth2[Dict[str, Any]]):
+    display_name = "Kakao"
+    logo_svg = LOGO_SVG
+
+    def __init__(
+        self,
+        client_id: str,
+        client_secret: str,
+        scopes: Optional[List[str]] = BASE_SCOPES,
+        name: str = "kakao",
+    ):
+        super().__init__(
+            client_id,
+            client_secret,
+            AUTHORIZE_ENDPOINT,
+            ACCESS_TOKEN_ENDPOINT,
+            refresh_token_endpoint=REFRESH_TOKEN_ENDPOINT,
+            revoke_token_endpoint=REVOKE_TOKEN_ENDPOINT,
+            name=name,
+            base_scopes=scopes,
+        )
+
+    async def get_id_email(self, token: str) -> Tuple[str, Optional[str]]:
+        async with self.get_httpx_client() as client:
+            response = await client.post(
+                PROFILE_ENDPOINT,
+                params={"property_keys": json.dumps(PROFILE_PROPERTIES)},
+                headers={**self.request_headers,
+                         "Authorization": f"Bearer {token}"},
+            )
+
+            if response.status_code >= 400:
+                raise GetIdEmailError(response.json())
+
+            account_info = cast(Dict[str, Any], response.json())
+            account_id = str(account_info.get('id'))
+            kakao_account = account_info.get('kakao_account')
+
+            try:
+                return account_id, kakao_account.get('email')
+            except:
+                return account_id, None
```

Comment:
This line misses coverage, but actually, it's not necessary.

When you call `.get` on a dictionary, it's already a safe method: if the key doesn't exist, it'll return `None`.

So you can directly do:

```py
return account_id, kakao_account.get('email')
```

without a `try`/`except`.

Reply:
woah i didn't know that, thanks for your information!

I will fix and make a commit.